src/metabase/api/search.clj

```diff
@@ -36,6 +37,22 @@
                respond
                raise))))
 
+(api/defendpoint POST "/force-reindex"
+  "If fulltext search is enabled, this will trigger a synchronous reindexing operation."
+  []
+  (api/check-superuser)
+  (cond
+    (not (public-settings/experimental-fulltext-search-enabled))
+    {:search-code 501, :message "Search index is not enabled."}
```

Comment:
I think this should be i18ned

Reply:
That's the wrong verb conjugation. `internationalized`

Reply:
i15d 🤣

Reply:
btw, what is search-code ? :D